phy/cluster/manual/clustering.py

```diff
@@ -6,15 +6,14 @@
 # Imports
 #------------------------------------------------------------------------------
 
-from collections import namedtuple, defaultdict, OrderedDict
-from copy import deepcopy
-
 import numpy as np
 
-from ...ext.six import iterkeys, itervalues, iteritems
 from ...utils.array import _as_array, _is_array_like
```

Comment:
why not using full name instead? 
I saw that a lot in phy, I dont understand the goal. the full qualified name is easier to grep/replace and is also more explicit.